pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -111,6 +111,7 @@ type zedagentContext struct {
 	rebootStack               string    // Previous reboot from nodeagent
 	rebootTime                time.Time // Previous reboot from nodeagent
 	restartCounter            uint32
+	rebootConfigCounter       uint32
```

Comment:
How about commenting these two lines to make it clear how they are different?

Reply:
Done..